### DISCUSSION TITLE
Changing splash screen on the fly

### DISCUSSION THREAD

**Discussion by OmkarK45** [zoontek/react-native-bootsplash#635] [Ideas]
First of all, thanks for the amazing work you do.

What if we could 
1. Show the default splash screen
2. Download a splash image from some remote source and save on device storage 
3. The next time app launches, we show this new splash image

Since you have the most context as far as native code is concerned, can you tell 
1. How feasible is this
2. What extent of customization is possible in this approach

**Comment by zoontek** [comment]
It cannot be done.

**Reply by OmkarK45** [reply]
Oh that's a bummer
Thanks for checking
